pkg/monitor/monitor/monitor_manager_test.go

```diff
@@ -383,7 +383,7 @@ func TestTidbMonitorSyncCreate(t *testing.T) {
 					},
 					Initializer: v1alpha1.InitializerSpec{
 						MonitorContainer: v1alpha1.MonitorContainer{
-							BaseImage: "pingcap/dm-monitor-initializer",
+							BaseImage: "pingcap/tidb-monitor-initializer",
```

Comment:
`pingcap/dm-monitor-initializer` isn't built after v6.0, and replace with `pingcap/tidb-monitor-initializer`